public/app/features/alerting/unified/components/rules/RuleDetailsActionButtons.tsx

```diff
@@ -29,6 +30,8 @@ export const RuleDetailsActionButtons: FC<Props> = ({ rule, rulesSource }) => {
   const rightButtons: JSX.Element[] = [];
 
   const { isEditable } = useIsRuleEditable(rulerRule);
+  const returnTo = location.pathname + location.search;
+  const isViewMode = inViewMode(location.pathname);
```

Comment:
I would like to do some refactoring so we basically could reuse the components without needing to check in what context it is being used. But I don't feel that I have time to solve that in the PR.